src/include/enclave/enclave_mem.h

```diff
@@ -29,13 +29,15 @@ void* enclave_mremap(
     size_t new_length,
     int mremap_fixed);
 
-int enclave_mmap_flags_supported(int flags, int fd);
+extern int mmap_files; // Allow MAP_PRIVATE or MAP_SHARED?
```

Comment:
Why not define `mmap_files` here?

Reply:
And make it common linkage?  I try to avoid common linkage whenever possible because it causes all manner of subtle linker behaviours.